### ISSUE TITLE
Can't sculpt the terrain

### ISSUE THREAD

**Issue by Eris7090** [Zylann/godot_heightmap_plugin#138]
**Describe the bug**
When I create a terrain I can't sculpt it.. Well I can but it will only raise up to a certain point then I can't do anything with it.  
**To Reproduce**
1. Create new godot project
2. Add the addons folder to the project & activate the addon in project settings 
3. add node then add HTerrain. 
4. add directory for terrain data
5. terrain appears but I can only raise it very very slightly and lower it very very slightly. like it's trapped in a box or something
If you see any errors in the console, that may also help.
 res://addons/zylann.hterrain/tools/normalmap_baker.gd:76 - Assertion failed.
every time I try to sculpt
**Expected behavior**
I expect to edit terrain like I can with your demo project. (yes, it works fine in the demo project)
**Screenshots**
https://i.imgur.com/Yb2SbLB.png
**Environment**
- OS: Arch
- Graphics card: Radeon™ Vega 8 Graphics
- Godot version: 3.2.1
- Plugin version: 1.1.1

**Comment by Zylann** [comment]
Are you using the GLES2 renderer?

**Comment by Eris7090** [comment]
Nah, I'm using GLES3. I was able to get it working if I installed it from the AssetLib. But I wanna figure out why it wasn't working by adding the plugin manually

**Comment by Zylann** [comment]
I tried in a fresh project by downloading current master, I could not reproduce it. Windows 10, nVidia GeForce GTX 1060 6GB/PCIe/SSE2 here

**Comment by FyreByrns** [comment]
I'm having the exact same issue. The plugin appears to freeze up when attempting to mouse over raised or lowered terrain, not moving the circle anywhere except on flat places.

**Comment by FyreByrns** [comment]
When I attempted to run my scene, a runtime error occurred at line 902 of hterrain_data.gd

**Comment by Zylann** [comment]
> I'm having the exact same issue. 

You are getting this when trying to raise the terrain? https://imgur.com/Yb2SbLB 
If you can still raise or lower the terrain more than 1 unit high, you may be having a different issue.

> When I attempted to run my scene, a runtime error occurred at line 902 of hterrain_data.gd

That line doesn't match a proper instruction in the last version of the plugin, you may try with the master branch.

**Comment by FyreByrns** [comment]
I downloaded the zip from master, but alright.

**Comment by Zylann** [comment]
@FyreByrns this is line 902: https://github.com/Zylann/godot_heightmap_plugin/blob/fb997981d5b40d1f572c2c89f1b3c0ef7fd0f1c0/addons/zylann.hterrain/hterrain_data.gd#L902
It's an `else`. No errors happen at an `else`, unless Godot reports the wrong line somehow?
Maybe master changed in the meantime? Having the actual error log would also hint at the proper location maybe.

**Comment by FyreByrns** [comment]
That's the same line it was reported as by godot, as soon as I can I'll get
the full log.

**Comment by Zylann** [comment]
There is something very peculiar going on with all-flat terrains. It eventually starts making hills normally but the first sculpting strokes seem to be struggling.
